Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -144,7 +144,7 @@ class TopTabsViewController: UIViewController {
         privateModeButton.snp.makeConstraints { make in
             make.centerY.equalTo(view)
             make.leading.equalTo(view).offset(10)
-            make.height.equalTo(view.snp.height)
+            make.height.equalTo(view.snp.height).offset(-8)
```

Comment:
I'm just wondering if it would make sense to create static constants for these kind of values? I'm not sure if there's an easy way to achieve that on iOS, but please don't take my comment too serious, I'm still learning :sweat_smile:

Reply:
It is indeed easily possible. Right now we have a mix and match of magic numbers and constants all over the place. I find it less invasive to FF code when I just change that number here without adding an extra line of code to define the constant. I must admit we're quite pragmatic there and don't have guidelines. Of course it makes sense when we have a number that's used in various places and which has dependencies. Then we tend to define constants.

Reply:
Thanks for the explanation! Yeah it makes sense. Btw, is this a FF class or is it ours? If it is a FF class, I'd agree on being pragmatic, but I wonder if we should also mark our changes on the classes that aren't ours or is this not a big danger for future upgrades?

Reply:
Agree with everything already said here 👍

Reply:
it's a FF class, so I'd vote for being less invasive. On the point of comments: We are less strict about this on iOS as the changes are very often obviously ours. So adding and maintaining comments means extra work. Also we don't upgrade as often as on Chromium where the comments are super beneficial. On iOS codebase is just less complex, thus we add comments where it makes sense e.g. bigger chunks, but not on every granular change. (... and we changed a lot)